tsup.config.ts

```diff
@@ -5,15 +5,16 @@ import { defineConfig } from 'tsup';
 
 export default defineConfig((options: Options) => ({
   ...options,
-  entry: ['src/index.ts'],
-  treeshake: true,
+
+  cjsInterop: true,
+  clean: true,
+  dts: true,
+  entry: ['src/index.ts', 'src/lib/configure-security.ts'],
   format: ['esm', 'cjs'],
-  silent: !options.watch,
   minify: false,
-  dts: true,
+  shims: true,
+  silent: !options.watch,
   sourcemap: true,
-  clean: true,
   splitting: true,
-  shims: true,
-  cjsInterop: true,
+  treeshake: true,
```

Comment:
per [their docs](https://tsup.egoist.dev/#tree-shaking) treeshaking is enabled by default but this flag defers to rollup for treeshaking. i'm fine with leaving this as is but it feels like bad DX that this flag exists in the first place.

Reply:
Agreed. I'm going to leave this in place as the rollup-generated dists are a lot cleaner.

#### without / with

![Screen Shot 2023-09-08 at 10 42 20 AM](https://github.com/readmeio/oas-to-har/assets/33762/464fb509-9c62-4f0a-9e0f-b7d13183d7d8)